src/librustc_builtin_macros/format.rs

```diff
@@ -410,7 +410,9 @@ impl<'a, 'b> Context<'a, 'b> {
                                 .iter()
                                 .filter(|fmt| fmt.precision_span.is_some())
                                 .count();
-                        e.span_label(span, &format!(
+                        e.span_label(
+                            span,
+                            &format!(
                             "this precision flag adds an extra required argument at position {}, \
```

Comment:
What happened here?

Reply:
I ran `x.py fmt` and the was modified like that. I could try to revert this?

Reply:
Ah I only use format-on-save so unrelated files aren't affected. But since it was `x.py fmt`, don't worry about taking it out.

Reply:
I had to rebase so i fixed it.